### DISCUSSION TITLE
Longhorn backups/restore

### DISCUSSION THREAD

**Discussion by AleksandarAtanackovic** [longhorn/longhorn#8591] [Q&A]
Hello!
I have a bucket on Minio that stores daily longhorn backups with retain number of 7. Today, if i delete all my backups and do a 1st backup(which is full backup i guess), and then do the backup everyday for the next 6 days, when i try to restore my backup after the 8th backup(the 1st backup is deleted on Minio) it doesn't work. I get failed to get object error, and when i check on Minio, that data block is really missing.
How can i fix this? 
There are similar #2393 questions, but they don't help.

**Comment by PhanLe1010** [comment]
>  when i try to restore my backup after the 8th backup(the 1st backup is deleted on Minio) it doesn't work

Hi @AleksandarAtanackovic Who deleted the 1st backup? Is it a Longhorn recurring job or something else?

**Reply by AleksandarAtanackovic** [reply]
Hi @PhanLe1010 , Minio storage deletes it. We configured it to have a retain period of 7 days so it automatically deletes all backups older than that. 
I tried setting the retain period of Longhorn backup to be less than 7 and now i need to wait and see what happens.
